src/main/java/gov/nasa/pds/tools/validate/rule/pds4/CollectionReferentialIntegrityRule.java

```diff
@@ -56,8 +59,12 @@
   
   private static final String LOGICAL_IDENTIFIER =
       "//*[starts-with(name(),'Identification_Area')]/logical_identifier";
+
+  private static final String VERSION_ID =
```

Comment:
@qchaupds this looks to be unused in this class

Reply:
The unused variable has been removed for next commit.